ush/python/pygfs/task/stage.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+import os
+from logging import getLogger
+from typing import Any, Dict, List
+
+from wxflow import (AttrDict, FileHandler, Task, cast_strdict_as_dtypedict,
+                    logit, parse_j2yaml, strftime,
+                    to_YMD, to_YMDH, Template, TemplateConstants)
+
+logger = getLogger(__name__.split('.')[-1])
+
+
+class Stage(Task):
+    """Task to stage initial conditions
+    """
+
+    @logit(logger, name="Stage")
+    def __init__(self, config: Dict[str, Any]) -> None:
+        """Constructor for the Stage task
+        The constructor is responsible for collecting necessary settings based on
+        the runtime options and RUN.
+
+        Parameters
+        ----------
+        config : Dict[str, Any]
+            Incoming configuration for the task from the environment
+
+        Returns
+        -------
+        None
+        """
+        super().__init__(config)
+
+        rotdir = self.task_config.ROTDIR + os.sep
+
+        self.task_config = AttrDict(**self.task_config)
+
+    @logit(logger)
+    def execute_stage(self, stage_dict: Dict[str, Any]) -> None:
+        """Perform local staging of initial condition files.
+        """
```

Comment:
Missing parameter and return lists. in the doc-block.

Reply:
My bad, added @ e14e6232, thanks!